dev/com.ibm.ws.org.eclipse.yasson.1.0/resources/yasson-messages.properties

```diff
@@ -0,0 +1,77 @@
+################################################################################
+# Copyright (c) 2016, 2018 Oracle and/or its affiliates. All rights reserved.
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v1.0 and Eclipse Distribution License v. 1.0
+# which accompanies this distribution.
+# The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+# and the Eclipse Distribution License is available at
+# http://www.eclipse.org/org/documents/edl-v10.php.
+#
+# Contributors:
+# Roman Grigoriadi
+# David Kral
+################################################################################
+
+cannotMarshallObject = Cannot marshall object
+typeResolutionError = Error resolving runtime type for type: {0}
+genericBoundNotFound = Generic bound not found for type {0} declared in {1}.
+cantConvertJsonValue = Can'''t convert JSON value into: {0}
+cantCreateRootInstance = Can'''t create root instance
```

Comment:
These messages sometimes use triple apostrophe (Can'''t), double apostrophe (Can''t), and single apostrophe (wasn't). Which of these is correct?